primitives/src/pcs/univariate_kzg/mod.rs

```diff
@@ -425,4 +518,68 @@ mod tests {
     fn batch_check_test() {
         batch_check_test_template::<Bls12_381>().expect("test failed for bls12-381");
     }
+
+    #[test]
+    fn test_multi_open() -> Result<(), PCSError> {
```

Comment:
Maybe add 1 more test for checking the `padded_len == poly_coeff.len()` case above.

Reply:
I'm already testing that (hopefully) by having: `let degree = rng.gen_range(5..max_degree / 2)`

To guarantee that we have cover that case, we might need to hardcode the degree we want to test, are you suggesting that?

Reply:
Ah OK, then no changes needed.

Reply:
as it turns out, your suggestion of explicitly testing those edge cases is a good one. 
i actually had a bug that didn't get detected. and fixed in [6336819](https://github.com/EspressoSystems/jellyfish/pull/231/commits/63368190989f72a86005a489a5840a4d38c5d706)